Enhancing enterprise permissions page (#3495)
fishtown-analytics/docs.getdbt.com: website/snippets/_enterprise-permissions-table.md

Key:

* (W)rite &mdash; Create new or modify existing. Includes `send`, `create`, `delete`, `allocate`, `modify`, and `read`.
* (R)ead &mdash; Can view but can not create or change any fields.

Permissions: 

* Account-level permissions &mdash; Permissions related to management of the dbt Cloud account. For example, billing and account settings.
* Project-level permissions &mdash; Permissions related to the projects in dbt Cloud. For example, repos and access to the IDE. 

### Account roles

> Account roles are primarily designed for managing the dbt Cloud account. Permissions to alter the account settings (for example, generating service tokens, inviting users, configuring SSO). They may also have access to project-level permissions as well. The **Account Admin** role is the highest level of access that can be assigned.  

Reviewer: I suggest we address the user directly so we can make these sentences active.
```suggestion
Account roles enable you to manage the dbt Cloud account and manage the account settings (for example, generating service tokens, inviting users, configuring SSO). They also provide project-level permissions. The **Account Admin** role is the highest level of access you can assign.  
```

| Environments            |       W       |               |       W         |                |   R    |
| Groups                  |       W       |               |       R         |       W        |   R    |
| Jobs                    |       W       |               |       W         |                |   R    |
| Licenses                |       W       |               |       W         |       W        |   R    |
| Metadata                |       R       |               |       R         |                |   R    |
| Permissions             |       W       |               |       W         |       W        |   R    |
| Profile                 |       W       |               |       W         |                |   R    |
| Projects                |       W       |               |       W         |       R        |   R    |
| Repositories            |       W       |               |       W         |                |   R    |
| Runs                    |       W       |               |       W         |                |   R    |
| Semantic Layer Config   |    W          |               |       W         |                |   R    |

### Project role permissions

> The project roles are intended for users who will be working within the projects in various capacities. They'll primarily have access to project-level permissions such as repos and the IDE, but may also have some account-level permisions.

Reviewer: ```suggestion
The project roles enable you to work within the projects in various capacities. They primarily provide access to project-level permissions such as repos and the IDE, but may also provide some account-level permissions.
```
